frontend/src/setupProxy.js

```diff
@@ -8,4 +8,11 @@ module.exports = (app) => {
       changeOrigin: true,
     })
   );
+  app.use(
+    "/llmwhisperer",
+    createProxyMiddleware({
+      target: process.env.REACT_APP_WHISPERER_PORTAL_URL,
+      changeOrigin: true,
+    })
+  );
```

Comment:
@jagadeeswaran-zipstack Can you expore and see if we can store this in the cloud repo and create a new instance if it's available?

Reply:
We can remove this from OSS and add a READ.me where we can inform devs to add this while they testing in local. because this is only required when we run the application in local.